db/migrate/20200224205220_add_columns_to_yellow_ribbon_program.rb

```diff
@@ -0,0 +1,35 @@
+class AddColumnsToYellowRibbonProgram < ActiveRecord::Migration[5.2]
```

Comment:
Please make a separate PR with just this migration and the `db/schema.rb` change which will be merged into master first.